docs-compose/overwatch_docs/content/DeployOverwatch/UCEDeployment/CloudStorageAccessRequirements/GCP.md

```diff
@@ -27,19 +27,37 @@ be provisioned read/write access to the storage target for the Overwatch Output
 (which will ultimately become your external location). This Service Account can be added to Overwatch Job/Interactive 
 Clusters.
 
-The following steps needs to be performed to external location’s GCS bucket permissions -
+The following steps needs to be performed in external location’s GCS bucket permissions -
+* Go to the `Permissions` tab of the external location’s GCS bucket
+* Click on `Grant Access`
+* Add Service Account which is attached to the Overwatch Job cluster in the `Add Pricipal` section
 * Add Service Account which is attached to the Overwatch Job cluster
 * Add Roles - 
   * Storage Admin 
   * Storage Legacy Bucket Owner
 
 ### Cluster Logging Locations Setup
+When you create a workspace, Databricks on Google Cloud creates two Google Cloud Storage GCS buckets in your GCP project:
+* `databricks-<workspace-id>` - stores system data that is generated as you use various Databricks features.
+This bucket includes notebook revisions, job run details, command results, and Spark logs
+* `databricks-<workspace-id>-system` - contains workspace’s root storage for the Databricks File System (DBFS). 
+Your DBFS root bucket is not intended for storage of production customer data.
```

Comment:
remove customer since the person reading this will be the customer

Reply:
done